frame.test.js

```diff
@@ -0,0 +1,115 @@
+const Frame = require('./frame')
+
+let frame = new Frame();
+
+describe("Frame", () => {
+
+  describe("#roll", () => {
+    it("should be able to record a roll", () => {
+      frame.roll(1);
+      expect(frame.getRolls()[0]).toBe(1);
+    })
+    it("should be able to record two rolls", () => {
+      frame.roll(2);
+      expect(frame.getRolls()[1]).toBe(2);
+    })
+  })
+
+  describe("#nonScoringRoll", () => {
+    it("should be able to roll a non-scoring roll (for bonus round at end)", () => {
+      frame = new Frame();
```

Comment:
all these tests look great but you have already defined frame, so perhaps you can remove this when refactoring

Reply:
Yeah I just needed to start a fresh frame for this test. I agree it's a bit verbose but wasn't sure how to do "before do" syntax in Javascript... will look into it!